pipeline/testing/symbolic/pilot.py

```
@@ -0,0 +1,331 @@
+            tool = "pynguin"
+            ok, tail = run_pynguin(target, pynguin_timeout)
+            pynguin_detail = {"version": version, "ok": ok, "output_tail": tail[-500:]}
+            if not ok:
+                tool = "static-approx"
+                proxy = True
+
+    static = static_approximation(target)
+    metrics = dict(static["metrics"])
+    metrics["tool"] = tool
+    metrics["proxy"] = proxy
+    # 求解超时率：proxy 模式不可测（null，显式 proxy=true）；pynguin 模式=运行超时占比
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)